commons-geometry-euclidean/src/test/java/org/apache/commons/geometry/euclidean/threed/Vector3DTest.java

```diff
@@ -1146,6 +1145,65 @@ void testEqualsAndHashCode_signedZeroConsistency() {
         Assertions.assertEquals(b.hashCode(), d.hashCode());
     }
 
+    @Test
+    void testHashCodeCollisions_symmetricAboutZero() {
+        final double any = Math.random();
+        final double neg = -any;
+        final double pos = +any;
+
+        final Vector3D negX = Vector3D.of(neg, 0.0, 0.0);
+        final Vector3D posX = Vector3D.of(pos, 0.0, 0.0);
+        final Vector3D negY = Vector3D.of(0.0, neg, 0.0);
+        final Vector3D posY = Vector3D.of(0.0, pos, 0.0);
+        final Vector3D negZ = Vector3D.of(0.0, 0.0, neg);
+        final Vector3D posZ = Vector3D.of(0.0, 0.0, pos);
+
+        int xNegHash = negX.hashCode();
+        int xPosHash = posX.hashCode();
+        int yNegHash = negY.hashCode();
+        int yPosHash = posY.hashCode();
+        int zNegHash = negZ.hashCode();
+        int zPosHash = posZ.hashCode();
+
+        String xMessage = String.format("negXHash: %s, posXHash: %s (expected to be non-equal)\n", xNegHash, xPosHash);
```

Comment:
These should be passed a suppliers to Assertions. However the Assertions will already have a statement containing the two values and that they were expected to be not equal. So we can just use:
```java
Assertions.assertNotEquals(zNegHash, zPosHash, "+/- z hash");
```